capitains_nautilus/cts/resolver.py

```diff
@@ -464,4 +432,84 @@ def clear(self):
         """ Deletes the database
         """
         clear_graph(self.graph_identifier)
-        super(SparqlAlchemyNautilusCTSResolver, self).clear()
+        super(_SparqlSharedResolver, self).clear()
+
+
+class SparqlAlchemyNautilusCTSResolver(_SparqlSharedResolver):
```

Comment:
`SparqlAlchemyNautilusCTSResolver` should probably be `SparqlAlchemyNautilusCtsResolver`